packages/vscode-extension-serverless-workflow-editor/src/extension/builtInVsCodeEditorSwfContributions.ts

```diff
@@ -66,6 +67,33 @@ export function setupBuiltInVsCodeEditorSwfContributions(args: {
     },
   };
 
+  const swfJsonValidationDiganosticsCollection: vscode.DiagnosticCollection =
+    vscode.languages.createDiagnosticCollection("swfValidation");
```

Comment:
```suggestion
  const swfJsonDiagnosticsCollection = vscode.languages.createDiagnosticCollection("SWF-JSON-VALIDATION-DIAGNOSTICS-COLLECTION");
```

Then extract this string to a constant, like `SWF_JSON_DIAGNOSTICS_COLLECTION_NAME` or something like that.